src/main/java/core/basesyntax/supplier/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.figure.enums.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    public Color getRandomColor() {
+        Random random = new Random();
```

Comment:
don't create Random object each time you call getRandomColor method
Meke it class-level variable